agent/containers/images/gen-tags-from-rpm

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+url_prefix="${1}"
+dist="${2}"
+arch="${3}"
+
+if [[ "${dist}" == "centos-8" ]]; then
+    dist="epel-8"
+elif [[ "${dist}" == "centos-7" ]]; then
+    dist="epel-7"
+fi
+
+version_string="$(yum list -q --repofrompath pbench,"${url_prefix}/pbench/${dist}-${arch}" pbench-agent.noarch 2>/dev/null | awk 'FNR==2{print $2}')"
```

Comment:
IIUC, this gets the version string making sure to look *only* in the designated repo - is that correct?

Reply:
Yes.  Adding a comment to make sure it is clear.